packages/cfd/src/Components/Routes/binary-routes.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Switch } from 'react-router-dom';
+import { PlatformContext } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import getRoutesConfig from '../../Constants/routes-config';
+import RouteWithSubRoutes from './route-with-sub-routes.jsx';
+
+const BinaryRoutes = props => {
+    const { is_dashboard } = React.useContext(PlatformContext);
+
+    return (
+        <React.Suspense
+            fallback={() => {
+                return (
+                    <div>
+                        <Localize i18n_default_text='Loading...' />
+                    </div>
```

Comment:
Are these divs necessary?

Reply:
Since fallback is required in react suspense, we are using plain text to avoid further issues.